src/steps/generic.rs

```diff
@@ -343,6 +343,25 @@ pub fn run_conda_update(ctx: &ExecutionContext) -> Result<()> {
         .status_checked()
 }
 
+pub fn run_mamba_update(ctx: &ExecutionContext) -> Result<()> {
+    let mamba = utils::require("mamba")?;
+
+    let output = Command::new("mamba")
+        .args(["config", "--show", "auto_activate_base"])
+        .output_checked_utf8()?;
+    debug!("Mamba output: {}", output.stdout);
+    if output.stdout.contains("False") {
+        return Err(SkipStep("auto_activate_base is set to False".to_string()).into());
+    }
+
+    print_separator("Mamba");
+
+    ctx.run_type()
+        .execute(mamba)
+        .args(["update", "--all", "-y"])
```

Comment:
The `-y` flag should only be enabled when `-y` flag is passed to `topgrade`.

<img width="818" alt="image" src="https://user-images.githubusercontent.com/28441561/229329190-6442c69f-3930-4239-8b1d-21c88ce53bdf.png">